src/styles/index.scss

```diff
@@ -1,3 +1,258 @@
+}
+
+.nav_link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: #060b35;
+  font-weight: 500;
+  height: 60px;
+  text-align: center;
+  margin-right: 50px;
+  text-transform: uppercase;
+  transition: color 0.3s ease;
+
+  &:hover {
+    color: #00acdc;
+  }
+
+  &.is-active {
+    color: #00acdc;
+
+    &.is-active::after {
+      content: '';
+      bottom: 0;
+      left: 0;
+      right: 0;
+      position: absolute;
+      background-color: #00acdc;
+      height: 4px;
+      border-radius: 8px;
+    }
+  }
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.nav_list .nav_link:nth-child(4):hover {
+  color: #00acdc;
+}
+
+/* #endregion */
+
+/* #region stars */
+.stars {
+  display: flex;
+  width: 100%;
+  max-width: 250px;
+  justify-content: flex-start;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-size: cover;
+    margin-right: 4px;
+  }
+
+  &--4 .stars__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+/* #endregion */
+
+/* #region main */
+main {
+  margin-top: 50px;
+}
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 46px;
+  margin-bottom: 48px;
+  justify-items: center;
+}
+
+.card {
+  width: 200px;
+  height: auto;
+  padding: 16px;
+  border-radius: 5px;
+  color: $main-text-color;
+  box-sizing: border-box;
+  transition:
+    transform 0.3s ease,
+    box-shadow 0.3s ease;
+  position: relative;
+
+  &:hover {
+    transform: scale(1.2);
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  }
+
+  &__title:hover {
+    color: #34568b;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__describe {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    margin-top: 40px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    text-align: center;
+
+    &__reviews {
+      color: $main-text-color;
+      font-size: px;
+      line-height: 14px;
+      width: 400px;
+      margin: 0;
+      text-align: right;
+    }
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    line-height: 18px;
+    height: 18px;
+    width: 100%;
+    margin-bottom: 16px;
+    margin-top: 24px;
+
+    &-text {
+      color: $secondary-color;
+      font-size: 12px;
+      margin: 0;
+    }
+
+    h2 {
+      font-weight: 700;
+      font-size: 16px;
+      margin: 0;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid $button-color;
+    border-radius: 5px;
+    width: 164px;
+    height: 38px;
+    color: $button-text;
```

Comment:
Ensure that the variable `$button-text` is defined in the imported `variables` file. If not, this will cause an error during compilation.